lib/src/matchers/jest_dom/util/get_value_of.dart

```diff
@@ -40,10 +38,10 @@ dynamic getValueOf(Element element, {dynamic Function(OptionElement option) getO
     getOptionValue ??= (option) => option.value;
     final selectedOptions = element.options.where((option) => option.selected);
     if (selectedOptions.isEmpty) {
-      return element.multiple ? const [] : null;
+      return element.multiple! ? const [] : null;
```

Comment:
Is this safe?

Reply:
I don't think it should be null, but I'll add an ArgumentError for a helpful error message